docs/index.md

```diff
@@ -4,60 +4,67 @@ id: Overview
 
 # Lind-Wasm
 
-Lind is a single-process sandbox that provides an option to safely execute programs. Lind executes applications using software fault isolation and a kernel microvisor to limit the potential of reaching bugs or security flaws in the application.
+Lind is a sandbox that isolates different applications in the same address space. Thus, conceptually, it executes different applications (which traditionally would be different processes) in separate, non-overlapping parts of a single address space, under a single, non-privileged Linux process.   To provide memory safety, control flow integrity, memory isolation, and similar properties, this version of Lind executes applications using WebAssembly for software fault isolation.  Lind also contains a custom kernel microvisor, written in Rust, to limit the potential damage of bugs or security flaws in an application.
 
-In Old Norse, Old High German and Old English a "lind" is a shield constructed with two layers of linden wood. Linden wood shields are lightweight, and do not split easily, an appropriate metaphor for a sandboxing system which employs two technologies.
+In Old Norse, Old High German and Old English a "lind" is a shield constructed with two layers of linden wood. Linden wood shields are lightweight, and do not split easily, an appropriate metaphor for a sandboxing system which is lightweight and which provides layered security.
 
 ## Core Concepts
 
-- **Cage**: Lightweight isolation boundary within a process
-    - Can run legacy code (may need recompilation)
-    - Protects and isolates memory
-- **Microvisor**: Small POSIX compliant kernel within a process
-    - Provides a POSIX interface
-    - Distinct isolation between cages
-- **3i (three eye)**: Capability-based POSIX interfaces between cages
+- **Cage**: This term describes the isolated memory namespace that an application executes in.  It is analogous to a process in Linux. 
+    - Can run legacy code compiled with Wasm as a target
+    - Protects and isolates memory, control flow, etc.
+- **Microvisor**: RawPOSIX is a small kernel running within the Lind process.  This is analogous to the Linux kernel.  Note, however, that this and all of the rest of Lind runs as an unprivileged Linux process.
+    - Provides a POSIXish interface (runs most Linux programs)
+    - Handles file descriptor separation, fork, exec, signals, threading, etc.
+- **3i (three eye)**: Capability-based POSIX interface to call between cages or into the microvisor.  This is conceptually similar to a programmable system call table and IPC interface. 
+    - Each cage has a separate system call table which can be independently changed to redirect into other cages or the microvisor
+    - Fast, isolated calling between cages
+    - Enables complex functionality (system call filtering, file systems, proxies, etc.) to be external to the microvisor
 
 ## Technology Overview
```

Comment:
I might switch the order of the Technology Overview and Components sections since this section seems to assume knowledge about the core components, so these should be listed first.